frontend/src/model/beta/releases/ModelReleaseReviewBanner.tsx

```diff
@@ -3,17 +3,25 @@ import Button from '@mui/material/Button'
 import Typography from '@mui/material/Grid'
 import Paper from '@mui/material/Paper'
 import { useTheme } from '@mui/material/styles'
+import { useGetReleasesForModelId } from 'actions/release'
 import { useState } from 'react'
+import { mutate } from 'swr'
 
+import { postReviewResponse } from '../../../../actions/review'
+import { ReleaseInterface } from '../../../../types/types'
 import ReviewWithComment, { ResponseTypeKeys } from '../../../common/ReviewWithComment'
 
 type ModelReleaseReviewBannerProps = {
   label: string
+  release: ReleaseInterface
+  mutateReviews: () => void
```

Comment:
Instead of passing this in as a prop, with no real safety on what's provided by the consumer of this component, could we instead do the following in this component?
```
const { mutateReviews } = useGetReviewRequestsForModel(release.modelId, release.semver, true)
const { mutateInactiveReviews } = useGetReviewRequestsForModel(release.modelId, release.semver, true)
```
We can then guarantee that we're calling the right mutate functions from this component, leaving no room for error for anyone using this component in future. It also allows us to call mutate from but active and inactive reviews.